crates/extricrate/src/lib.rs

```diff
@@ -1,36 +1,56 @@
 #![allow(dead_code, unused_variables)]
 
 pub mod dependencies {
-    use std::collections::HashMap;
+    use std::collections::{HashMap, HashSet, VecDeque};
+    use std::fs::read_to_string;
+    use std::path::{Path, PathBuf};
 
+    use quote::ToTokens;
+    use syn::visit::{self, Visit};
+    use syn::{
+        Ident, ItemUse, UseGlob, UseGroup, UseName, UsePath, UseRename, UseTree, parse_file,
+    };
+    use thiserror::Error;
+
+    #[derive(Debug, PartialEq, Eq)]
     pub struct ModuleName(String);
+    impl ModuleName {
+        fn new(name: String) -> Self {
+            Self(name)
+        }
+    }
 
+    #[derive(Debug, PartialEq, Eq)]
     pub struct Position {
         line: u32,
         col: u32,
     }
 
+    #[derive(Debug, PartialEq, Eq)]
     pub struct Extent {
         start: Position,
         // Inclusive
         end: Position,
     }
 
+    /// A single, separate use statement.
+    #[derive(Debug, PartialEq, Eq)]
+    pub struct NormalizedUseStatement {
```

Comment:
Just a note, nothing to change here: when normalizing use statements, we'll need to handle "super"